README.md

```diff
@@ -158,3 +158,18 @@ export default MyComponent;
 > Although we have just higlighted two combinations of how to load and use the
 > web component, it should be possible to use this library with all the major
 > frontend frameworks.
+
+## Migrating from v1 to v2
```

Comment:
I think it worth to add a link to this paragraph in the v2.0.0 release's description.